packages/web-app-files/src/composables/actions/files/useFileActionsCreateNewFolder.ts

```diff
@@ -66,12 +68,14 @@ export const useFileActionsCreateNewFolder = ({
         resource.indicators = getIndicators({ resource, ancestorMetaData: unref(ancestorMetaData) })
       }
 
-      store.commit('Files/UPSERT_RESOURCE', resource)
+      await store.commit('Files/UPSERT_RESOURCE', resource)
```

Comment:
My IDE suggests there's no difference when using `await` here, however the scrolling only happens if you put it there 🤷🏽

Reply:
I don't get that either... `commit` is a synchronous call / doesn't return a promise, so `await` makes no sense at all...

Reply:
https://stackoverflow.com/a/55263084

What you actually want is probably `await nextTick()`

Reply:
Done, thx for the hint!